src/constants/grid.js

```diff
@@ -18,8 +18,23 @@ const getGridExtent = () => {
 
 const EARTH_CIRCUIT = 40075000 //in meters
 
+//size of tile in pixels
+const PIXEL_TILE_SIZE = 250;
+
+//Resolutions
+//Resolution is number of meters displayed in one pixel (m/px).
+
+// Maximum resolution on equator. 
+const maxResolution = EARTH_CIRCUIT / PIXEL_TILE_SIZE;
+const resolutions = []
+const levels = 25;
+for (let i = 0; i < levels; ++i) {
+	resolutions[i] = maxResolution / Math.pow(2, i + 1);
```

Comment:
With respect to the previous comment
```suggestion
	resolutions[i] = maxResolution / Math.pow(2, i);
```